kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowRunnerViewModel.kt

```diff
@@ -76,7 +71,10 @@ internal class WorkflowRunnerViewModel<OutputT : Any>(
         snapshot
     ) { renderings, outputs ->
       @Suppress("UNCHECKED_CAST")
-      WorkflowRunnerViewModel(viewRegistry, renderings, outputs, this) as T
+      WorkflowRunnerViewModel(
+          // TODO Pulling the coroutineContext out of the scope like this is sleazy, isn't it?
```

Comment:
Nope, why?

Reply:
Phew. Scared by "for advanced use only" kdoc on it. I haven't internalized structured concurrency yet.

Reply:
I never noticed that kdoc…